src/harness/testcases/relinquishment_testcase.py

```diff
@@ -123,3 +124,25 @@ def test_10_13_4_1_2(self):
       self.assertEqual(response['cbsdId'], cbsd_id)
       self.assertEqual(response['grantId'], grant_id[i])
       self.assertEqual(response['response']['responseCode'], 0)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_RLQ_4(self):
+    """CBSD relinquishment request with CBSD ID that does not exist in SAS.
+
+    CBSD Harness sends Relinquishment Request to SAS including CBSD ID and
+    Grant ID in correct format but the CBSD ID does not exist in SAS.
+    The response should be FAIL.
+    """
+
+    # Relinquish the grant
+    request = {
+        'relinquishmentRequest': [{
+            'cbsdId': 'A non-exist cbsd id',
+            'grantId': 'A non-exist grant id'
+        }]
+    }
+    response = self._sas.Relinquishment(request)['relinquishmentResponse'][0]
+    # Check the relinquishment response
```

Comment:
The text of the test says we need to check that grantId matches the grantId in the request but the SAS-CBSD TS says grantId should only be included if it's a valid grantId. It seems that your code is correct and that the test text needs to be updated; do you agree?